### PR TITLE
Add new updated_by/on fields

### PR DESCRIPTION
**Pull request by hectorcorrea**
Script for Robert Knight to update the schema in td-meta1

Emailed tigerdata distribution list. Waiting on confirmation.

See also https://github.com/pulibrary/tiger-data-app/pull/291

### CODE REVIEW

**Comment by hectorcorrea** [comment]
This has been applied to td-meta1 and it's ready to be merged

File: docs/schema_script.txt [hunk 1]
```diff
@@ -20,4 +20,7 @@ asset.doc.type.update :create true :description "Project metadata" :type tigerda
     :element -name data_users_ro -type string -index true -min-occurs 0               -label "A person who has read-only access privileges to the project" \
     :element -name departments   -type string -index true -min-occurs 1               -label "The primary Princeton University department(s) affiliated with the project" \
     :element -name created_on    -type date               -min-occurs 1 -max-occurs 1 -label "Timestamp project was created" \
-    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" >
+    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" \
+    :element -name updated_on    -type date               -min-occurs 0 -max-occurs 1 -label "Timestamp project was updated" \
+    :element -name updated_by    -type string             -min-occurs 0 -max-occurs 1 -label "User that created the created" >
```
Comment (bess) [inline docs/schema_script.txt:25]: Is this label right? Maybe it's supposed to read "User that updated the project"?

**Review by bess — CHANGES_REQUESTED** [review]
@hectorcorrea This looks great except the last label seems confusing.

File: docs/schema_script.txt (hunk 1 above)
Comment (hectorcorrea) [reply docs/schema_script.txt:25]: "created the created" -- yeah, of course that's what I meant 🤦🏼 -- I'll fix it 😄

**Review by bess — APPROVED** [review]

**Pull request merged**